h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -972,7 +976,9 @@ void defaultGBMs(Key<Grid> gridKey) {
     } else {
       stackedEnsembleParameters._metalearner_nfolds = buildSpec.build_control.nfolds;
     }
-    // TODO: Add fold_assignment
+
+    stackedEnsembleParameters._keep_cross_validation_fold_assignment = buildSpec.build_control.keep_cross_validation_fold_assignment;
```

Comment:
The Stacked Ensemble model does not actually have a `keep_cross_validation_fold_assignment` argument yet, so I just added a JIRA for it: https://0xdata.atlassian.net/browse/PUBDEV-5670  

We could add that in a separate PR before we merge this one, or we could add it afterwards (though it probably makes more sense to add it prior to adding it to AutoML).

Reply:
This is a generic model builder argument. So it should be there.

Reply:
I would test this scenario out @deil87

Reply:
Ok, good.  That means that we probably just never exposed this arg in R or Python, but that it's probably functional in Java, so your code should work and we don't need to fix any SE code prior to merging this. :-)

Reply:
It exists. Compiler would throw an exception to me otherwise.